components/text-toast.json

```diff
@@ -0,0 +1,15 @@
+{
+  "textToast": {
```

Comment:
if the colors used for desktop differ from the ones in mobile (see https://www.figma.com/file/z6oi9Ku3X7rM1Wj2451B9u/Components---Android?node-id=6484%3A25110), please change the location of this file to platformcomponents/desktop

Reply:
they do - will change

Reply:
@nataliadelmar changed - pls check again